case/Input/tube.in

```diff
@@ -2,19 +2,19 @@
 1.E-3                   ! eps_Ewld
 8                       ! PBspln_Ewld
 
-1                       ! nCellTypes
-1                       ! viscRat
-1.                      ! refRad
+3                       ! nCellTypes
+1 1 1                   ! viscRat
+1. 1. 1.                ! refRad
```

Comment:
this doesn't seem right, shoudn't it be 1 number? what about viscRat? Regardless, please create multiple "cases" that serve different example purposes.

Reply:
See `ModConf.F90` line 245:

`read(funit, *) nCellTypes;        print *, 'nCellTypes = ',nCelltypes
      read(funit, *) viscRatIN(1:nCellTypes)
      print *, 'viscRat = ', viscRatIN(1:nCellTypes)
      read(funit, *) refRadIN(1:nCellTypes)
      print *, 'refRad = ', refRadIN(1:nCellTypes)`

Yes, I will move this sickle+rbc simulation to a new case folder.